app/src/configurable_auth.h

```diff
@@ -27,6 +28,10 @@ namespace scitt
       auto identity = JwtAuthnPolicy::authenticate(tx, ctx, error_reason);
       if (!identity)
       {
+        SCITT_INFO(
```

Comment:
This code path would also be reached when the configuration allows unauthenticated access (then the `ConfigurableEmptyAuthnPolicy` would succeed), so the check for that (see below) has to be moved up, otherwise we log errors when there haven't been any.

Reply:
It seem the ConfigurableEmptyAuthnPolicy is cehck before ConfigurableJwtAuthnPolicy.
see main.cpp 282
const ccf::AuthnPolicies authn_policy = {
        std::make_shared<ConfigurableEmptyAuthnPolicy>(),
        std::make_shared<ConfigurableJwtAuthnPolicy>(),
      };
According to the docs in ccf, any single successful check is sufficient to grant access. And it seem the check is in order of the authn_policy vector.

Reply:
You're right, the order matters of course.